language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/projectls/defmodsource3.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/module1;
```

Comment:
Missing the license header?

Reply:
In the LS test cases we haven't put the license header, as it would be syntactically incorrect.